modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/index.js

```diff
@@ -76,7 +76,10 @@ export {App as DataSet};
 
 // Renderers API
 
-export {default as DataRenderers} from './data_renderers/index';
+export {
+	dataRenderers as DataRenderers,
+	inputRenderers as InputRenderers,
+} from './data_renderers/index';
```

Comment:
@oliv-yu @FabioDiegoMastrorilli  If we remove `DataRenderers` export, won't this break [this import in commerce-js-web](https://github.com/liferay/liferay-portal/blob/master/modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/dataRenderers.js#L15)? It looks like we are assuming renderers are in a merged object there.

Reply:
Nice spot @markocikos :) At the moment we only import views-related renders not to duplicate rendering logic.

The InputRenderers were written specifically for the dataset display and there's no actual point in exporting them out of the dataset module (at least from what I remember - It was some time ago)

Reply:
I see. So, we are safe to update `DataRenderers` to `dataRenderers` in `commerce-js-web`?

Reply:
I don't know if I got the question, but we are still exporting `DataRenderers`, no? 

The only difference is that the exported renderers will only be pure views, not inputs (which we don't need)

Reply:
Oh yeah, you are right. Then, we are good to go!